lighthouse-core/test/lib/minification-estimator-test.js

```diff
@@ -180,6 +180,23 @@ describe('minification estimator', () => {
       assert.equal(computeJSTokenLength(js), 9);
     });
 
+    it('should handle regular expression character classes', () => {
+      const js = `
+        /regex [^/]\\//.test('this should be in string not comment 123456789')
```

Comment:
Is this valid?
```javascript
// Uncaught SyntaxError: Unexpected token .
/regex [^/]\\//.test('this should be in string not comment 123456789')
```
Should it be:
```javascript
/regex [^/]\\/.test('this should be in string not comment 123456789')
```
Or if you're trying to include a double / then:
```javascript
/regex [^/]\\\//.test('this should be in string not comment 123456789')
```

Reply:
Yeah I'm open to ideas to make this clearer. Basically it's the JS equivalent of

```js
/regex [^/]\//.test('this should be in string not comment 123456789')
```

Because we're sticking the `\` inside a string we need an extra `\` to escape it, but we don't need to escape anything else

Reply:
FWIW, I think it's much clearer in vscode which does the highlighting to show what's being escaped, so maybe just add my js example to the comments?